bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/read/ScanAdapter.java

```diff
@@ -261,4 +265,22 @@ private long convertUnits(long hbaseUnits) {
     }
     return Optional.<Filters.Filter>of(interleave);
   }
+
+  private Optional<Filters.Filter> createColFamilyTimeRange(Scan scan) {
+    if (scan.getColumnFamilyTimeRange().isEmpty()) {
+      return Optional.absent();
+    }
+    // Builds filter of the form
+    // ("family1" | "rangeStart, rangeEnd") + ("family2" | "rangeStart2, rangeEnd2")
```

Comment:
This notation is confusing `|` usually means OR. It would be easier to read if you use `|` for interleave and `&` for intersection

Reply:
I have addressed this comment, please have a look.